src/Database/Esqueleto/Internal/Internal.hs

```diff
@@ -1426,11 +1426,15 @@ data LockingKind where
       -- ^ @LOCK IN SHARE MODE@ syntax.  Supported by MySQL.
       --
       -- @since 2.2.7
-    ForUpdateOfSkipLocked :: PersistEntity val => [(SqlExpr (Entity val))] -> LockingKind
+    ForUpdateOfSkipLocked :: [LockableEntity] -> LockingKind
       -- ^ @FOR UPDATE OF tablename SKIP LOCKED@ syntax. Supported by MySQL, and PostgreSQL
       --
       -- @since 3.5.4.2
 
+-- | Existential type to wrap table type entities for LockingKind
```

Comment:
This is necessary since a `ForUpdateOfSkipLocked` can take a list of different types